build.sbt

```diff
@@ -115,7 +116,21 @@ lazy val server = (project in file("server")) enablePlugins(JavaAppPackaging) se
   ),
   Compile / PB.targets := Seq(
     scalapb.gen() -> (Compile / sourceManaged).value / "scalapb"
-  )
+  ),
+  daemonUser in Docker := "root",
```

Comment:
The docker:publishLocal outputs a fine docker image that doesn't run as root. I agree with the enhancements in the documentation but don't understand the need to change the user, and add the setcap. Could you please clarify why the defaults don't work for you?

Reply:
setcap was needed so that the server would be able to use privileged ports like 80 inside the container. But you're right that we don't need to keep the user as root after setcap.

Reply:
A clear. I chose port 8000 in my config file so the current docker container runs fine for me. We could do without the setcap if we make sure the default config file in the repo has a different port (not 80). That would be my preference to keep things as simple as possible. Just my 2cts

Reply:
@krisgeus Good point. I just did some research about [the potential risks](https://medium.com/jobteaser-dev-team/docker-user-best-practices-a8d2ca5205f4) of using root in a container. Totally agree that we should not use root.

@FX196 could you just change the server config template to use 8080 instead?